### PR TITLE
Update classList regarding FF version

### PR DESCRIPTION
**Pull request by rlbabyuk**

### CODE REVIEW

File: src/widgetastic/browser.py [hunk 1]
```diff
@@ -564,12 +568,16 @@ def classes(self, locator, *args, **kwargs):
...
-        self.logger.debug('css classes for %r => %r', locator, result)
-        return result
+            if self.browser_version < '46':
+                command = 'return arguments[0].classList;'
+            else:
+                command = 'return arguments[0].classList.value;'
+            script_run = self.execute_script(
+                command, self.element(locator, *args, **kwargs),
+                silent=True)
+            result = set(script_run.split(' ')) if isinstance(script_run, str) else set(script_run)
```
Comment (mfalesni) [inline src/widgetastic/browser.py]: I would probably go for `re.split` with `"\s+"` as the regexp. Just to be on the safe side.

File: src/widgetastic/browser.py [hunk 2]
```diff
@@ -564,12 +568,16 @@ def classes(self, locator, *args, **kwargs):
...
-            command = 'return arguments[0].classList;'
-        result = set(self.execute_script(
-            command, self.element(locator, *args, **kwargs),
-            silent=True))
-        self.logger.debug('css classes for %r => %r', locator, result)
-        return result
+            if self.browser_version < '46':
+                command = 'return arguments[0].classList;'
+            else:
+                command = 'return arguments[0].classList.value;'
```
Comment (mfalesni) [inline src/widgetastic/browser.py]: Can you please add an explanation why is this happening?

Btw. if this is for firefox, I don't see ifs for firefox :)

File: src/widgetastic/browser.py [hunk 3]
```diff
@@ -564,12 +572,18 @@ def classes(self, locator, *args, **kwargs):
...
-        return result
+            # js classList call was changed for ff starting 46 version
+            if self.browser_version <= 45 and self.browser_type == 'firefox':
+                command = 'return arguments[0].classList;'
+            else:
+                command = 'return arguments[0].classList.value;'
+            script_run = self.execute_script(
+                command, self.element(locator, *args, **kwargs),
+                silent=True)
+            result = (set(re.split("\s+", script_run)) if isinstance(script_run, basestring)
```
Comment (mfalesni) [inline src/widgetastic/browser.py]: **REQUIRED**: replace basestring with six.string_types

Comment (rlbabyuk) [reply src/widgetastic/browser.py]: done

**Pull request merged**